docs/FAQ/FAQ-LightningNetwork.md

```diff
@@ -334,6 +335,24 @@ export LIGHTNING_ALIAS="namehere"
 . ./btcpay-setup.sh -i
 ```
 
+### How to install ThunderHub?
+To install ThunderHub on your instance apply the following:
+
+```
+export BTCPAYGEN_ADDITIONAL_FRAGMENTS="\$BTCPAYGEN_ADDITIONAL_FRAGMENTS;opt-add-thunderhub"
+
+. btcpay-setup.sh -i
+```
+
+If you get the following warning message **Unable to connect to this node** apply the following:
+
```

Comment:
I think it would be better if you can mention the reason of the error or why the following commands are a solution.

Reply:
Good point! added a more descriptive reason

Reply:
Don't know if I should also add which domain it is but that goes a bit deeper into the docker compose setup which is maybe not needed?